qiskit_ibm_provider/ibm_backend.py

```diff
@@ -459,6 +459,11 @@ def run(
         # pylint: disable=arguments-differ
         validate_job_tags(job_tags, IBMBackendValueError)
 
+        if hasattr(self._api_client, "backend_status"):
```

Comment:
Still not sure if I understand why this check is needed... is there a scenario where _api_client will not have backend_status?

Reply:
Yeah when unit testing the job states the fake api's don't have `backend_status`

The tests fail [here](https://github.com/Qiskit/qiskit-ibm-provider/runs/6218410403?check_suite_focus=true) without this check

Reply:
Then I guess the right way to do it would be to add `backend_status` method in `BaseFakeAPI`.

Reply:
Fixed!

Reply:
Tests are failing, I think it should be in BaseFakeAccountClient https://github.com/kt474/qiskit-ibm/runs/6312293752?check_suite_focus=true#step:5:1804